services/common/src/main/java/com/linkedin/openhouse/common/stats/model/RetentionStatsSchema.java

```diff
@@ -0,0 +1,22 @@
+package com.linkedin.openhouse.common.stats.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/** Data Model for capturing retention stats about a table. */
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder(toBuilder = true)
+public class RetentionStatsSchema {
```

Comment:
Considering RetentionConfig is already defined with same fields. Can we restructure to avoid duplication?
Maybe something like this:
```java
RetentionStats {
  protected final String columnName;
  protected final String columnPattern;
  protected final Retention.GranularityEnum granularity;
  protected final int count;
}

and  then have 

RetentionConfig extends RetentionStats {}

``` 

This would preserve RetentionConfig while allowing extensibility to add other fields in retentionConfig without affecting RetentionStats